crates/bevy_camera/src/primitives.rs

```diff
@@ -45,8 +45,14 @@ impl MeshAabb for Mesh {
 ///   or `Sprite` component,
 /// - don't have the [`NoFrustumCulling`] component.
 ///
-/// It won't be updated automatically if the space occupied by the entity changes,
-/// for example if the vertex positions of a [`Mesh3d`] are updated.
+/// It won't be updated automatically if the space occupied by the entity
+/// changes, for example if the vertex positions of a [`Mesh3d`] are updated, or
```

Comment:
I think this comment is out of date after https://github.com/bevyengine/bevy/pull/18742